.env.test

```diff
@@ -3,10 +3,11 @@ SERVER_WEBSOCKET_CHECK_ORIGIN="true"
 SERVER_WEBSOCKET_MAX_CONN="30000"
 SERVER_WEBSOCKET_READ_BUFFER_SIZE="10240"
 SERVER_WEBSOCKET_WRITE_BUFFER_SIZE="10240"
-SERVER_WEBSOCKET_CONN_UNIQ_ID_HEADER="x-user-id"
-SERVER_WEBSOCKET_PING_INTERVAL_MS=30000
-SERVER_WEBSOCKET_PONG_WAIT_INTERVAL_MS=60000
-SERVER_WEBSOCKET_WRITE_WAIT_INTERVAL_MS=5000
+SERVER_WEBSOCKET_CONN_ID_HEADER="x-user-id"
+SERVER_WEBSOCKET_CONN_TYPE_HEADER="x-user-type"
```

Comment:
Type is confusing. Shall we use group here?

Reply:
Ya actually. Connection type could mean something else. Group sounds good.